jota/src/main/java/jota/IotaAPI.java

```diff
@@ -933,6 +916,20 @@ public Bundle traverseBundle(String trunkTx, String bundleHash, Bundle bundle) t
             throw new InvalidBundleException();
         }
 
+
+        // If message or tag is not supplied, provide it
+        // Also remove the checksum of the address if it's there
+
+        for (Transfer transfer : transfers) {
+
+            transfer.setMessage(transfer.getMessage().isEmpty() ? StringUtils.rightPad(transfer.getMessage(), 2187, '9') : transfer.getMessage());
```

Comment:
could we move these (`2187`, `27`) to constants?

Reply:
what if `getMessage()` is `null`?

Reply:
The List<Transfers> is validated before. I've added another constructor to the Transfers.class.